### PR TITLE
Fix truncation warnings in tokenizer

### PR DESCRIPTION
**Pull request by tholor**
Solving #525

### CODE REVIEW

File: farm/data_handler/input_features.py [hunk 1]
```diff
@@ -43,6 +43,8 @@ def sample_to_features_text(
         # This should change once we git rid of FARM's tokenize_with_metadata()
         inputs = tokenizer(text,
                            return_token_type_ids=True,
+                           truncation=True,
+                           truncation_strategy="longest_first",
```
Comment (Timoeller) [inline farm/data_handler/input_features.py:47]: Why do we want to have truncation=True when we set it to false in line 65?
Also truncation_strategy is deprecated and has been removed in earlier commits. Is it necessary now?

Are we not doing truncation outside of encoding?

**Review by Timoeller — COMMENTED** [review]
Posed some questions as comment

File: farm/data_handler/input_features.py (hunk 1 above)
Comment (tholor) [reply farm/data_handler/input_features.py:47]: For the fast tokenizer we feed here the whole untruncated text again (therefore we need truncation). In line 65 we use the slow tokenizer (which just gets the truncated tokens)

**Review by Timoeller — APPROVED** [review]
looking good

**Comment by PhilipMay** [comment]
Thanks. :-)

**Pull request merged**